### DISCUSSION TITLE
Discontinuous plot in time domain simulation

### DISCUSSION THREAD

**Discussion by seedpower** [CURENT/andes#381] [Q&A] [answered]
```
import andes

ss = andes.run(andes.get_case("ieee14/ieee14_full.xlsx"), setup=False, default_config=True)

ss.add("Fault", dict(bus=9, tf=1.0, tc=1.5))
ss.setup()
ss.PFlow.run()

# tf = 2.0s
# ss.TDS.config.tf = 2.0
# ss.TDS.run()

# tf = 3.0s
ss.TDS.config.tf = 3.0
ss.TDS.run()
ss.TDS.load_plotter()
ss.TDS.plotter.plot(ss.GENROU.delta)

```
I got the picture below:
![image](https://user-images.githubusercontent.com/11022830/201816384-03f9d726-7c43-458b-8be7-c70a0afaf18d.png)

Now, if I uncomment tf = 2.0s code
```
# tf = 2.0s
ss.TDS.config.tf = 2.0
ss.TDS.run()
```
This picture is :
![image](https://user-images.githubusercontent.com/11022830/201816451-a4f6b296-1742-412b-8a66-7e85e4c94651.png)

Compared with them, they look a little discontinuous at t = 2.0s.

When I turn `ss.TDS.config.tstep` down，the discontinuity between them would be smaller. But it will take more time. 

Q1: Whether this discontinuity will affect the time-domain simulation results?

Q2: Is there a better way to avoid this discontinuity without modifying `ss.TDS.config.tstep`?

**Comment by cuihantao** [comment]
This seems to be an issue in how the program resumes a simulation. The two are probably off by one step. Let me check that and come up with a fix.

**Reply by seedpower** [reply]
Please fix it as soon as possible. I need the diagram made by it. I'm looking forward to it.

**Reply by cuihantao** [reply]
I'll work on it over the weekend.

**Comment by cuihantao** [comment, marked as answer]
Issue fixed in https://github.com/cuihantao/andes/commit/8f17167872f9c4874a352cad68b093d703903bea. Thank you for reporting.

Mind sharing what project you are using ANDES for?

**Reply by seedpower** [reply]
Thanks for your repair, it works very well. I use ANDES for reinforcement learning. The software is great.
